packages/sdk/src/models/userinfo.ts

```diff
@@ -28,3 +28,19 @@ export interface UserInfo {
    */
   preferredUserName: string;
 }
+
+export interface UserTenantIdAndLoginHint {
```

Comment:
can we merge this interface into `UserInfo`? we just need to add tenantId, the preferredUserName can be used as loginHint (please double confirm)

Reply:
Will not merge according to offline discuss